torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -761,7 +761,8 @@ def benchmark_all_configs(self, *args, **kwargs):
             "CachingAutotuner.benchmark_all_configs",
             log_pt2_compile_event=True,
             metadata={"kernel_name": self.inductor_meta.get("kernel_name")},
-            dynamo_compile_runtime_column_us="runtime_triton_autotune_time_us",
+            # TODO(masnesral): Enable this when we figure out how to get the CompileId:
+            # dynamo_compile_runtime_column_us="runtime_triton_autotune_time_us",
```

Comment:
In the end, we only really care about disabling it for tlparse due to the unnecessary spew, so you could have also just turned it off on that side and kept the dynamo_compile rows. But this is fine